modules/seqexec/web/client/src/main/scala/seqexec/web/client/circuit/StatusAndLoadedSequencesFocus.scala

```diff
@@ -22,34 +21,57 @@ final case class SequenceInSessionQueue(id:            Observation.Id,
                                         active:        Boolean,
                                         loaded:        Boolean,
                                         name:          String,
-                                        targetName:    Option[TargetName],
+                                        targetName:    Option[String],
```

Comment:
Why the change from `TargetName` to `String`?

Reply:
it was trying to fix a bug but i'll revert it